Release/machine/SIRIUS/common/+sirius_excdata/save_excdata.m

```diff
@@ -86,7 +86,7 @@ function save_excdata(sorting, fname, currents, main_mpole, harmonics, n_avg, s_
 '# corrector-vertical      | VKick > 0      | BXL  < 0    | -1.0        | I > 0';
 '# quadrupole (focusing)   | KL    > 0      | D1NL < 0    | -1.0        | I > 0';
 '# quadrupole (defocusing) | KL    < 0      | D1NL > 0    | -1.0        | I > 0';
-'# quadrupole (skew)       | KL    > 0      | D1SL > 0    | +1.0        | I > 0';
+'# quadrupole (skew)       | KsL   < 0      | D1SL > 0    | -1.0        | I > 0';
```

Comment:
See my comment in the other related PR:

https://github.com/lnls-sirius/control-system-constants/pull/114#discussion_r329322754

Reply:
Ok @xresende , I changed the notation back to KL.